tests/conftest.py

```diff
@@ -3,6 +3,8 @@
 
 # Pytest fixtures and redefined-outer-name don't mix well. Disable it.
 # pylint:disable=redefined-outer-name
+# We import some fixtures that are unused. Disable that too.
+# pylint:disable=unused-import
```

Comment:
Why do we import unused fixtures?

Reply:
`conftest.py` is where you can share fixtures with the rest of the tests. That way you don't have to import them manually in each test.